recipe/0540-link-for-opening-multiple-canvases/annotation.json

```
@@ -0,0 +1,28 @@
+{
+   "@context": "http://iiif.io/api/presentation/3/context.json",
+   "id": "{{ id.url }}",
+   "type": "Annotation",
+   "motivation": ["contentState"],
+   "target": [
+       {
+           "id": "https://iiif.io/api/cookbook/recipe/0318-navPlace-navDate/canvas/2",
+           "type": "Canvas",
+           "partOf": [
+               {
+                   "id": "https://iiif.io/api/cookbook/recipe/0318-navPlace-navDate/manifest-2.json",
+                   "type": "Manifest"
+               }
+           ]
+       },
+       {
+           "id": "{{ id.path }}/canvas/p2",
+           "type": "Canvas",
+           "partOf": [
+               {
+                   "id": "{{ id.path }}/manifest",
```

This results in a 404 and this property in the json:

`"id": "https://preview.iiif.io/cookbook/0540-link-for-opening-multiple-canvases/recipe/0540-link-for-opening-multiple-canvases/canvas/p2",`

I think we need this:

```suggestion
                   "id": "{{ id.path }}/manifest.json",
```